conda_build_all/version_matrix.py

```diff
@@ -72,6 +73,41 @@ def conda_special_versions(meta, index, version_matrix=None):
             yield case
 
 
+def parse_specifications(requirements):
+    """
+    Parse a list of specifications, turning multi-line specifications into
+    a single specification.
+    """
+    requirement_specs = defaultdict(list)
+    # Generate a list of requirements for each spec name to ensure that
+    # multi-line specs are handled.
+    for spec in requirements:
+        spec_details = spec.split()
```

Comment:
Maybe it just makes sense to do a ``split(1)`` as we don't care about the difference between the 2 and 3 part cases.